### ISSUE TITLE
Allow to return detections not associated with an animal

### ISSUE THREAD

**Issue by peterdesmet** [inbo/etn#186]
If I remember correctly, this is a request made by @jreubens.

**Comment by jreubens** [comment]
Yes, that is a request from my side. Aubri is working on a view to enable this. Once available @aubrivliz can provide you the view.
